### PR TITLE
{cae}[foss/2019a] Gerris v20131206

### PR DESCRIPTION
**Pull request by soheil-soltani**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
@soheil-soltani can you add the missing dependency (or, if they are in another open PR, mention it here)?

File: easybuild/easyconfigs/g/Gerris/Gerris-20131206-foss-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+homepage = 'http://gfs.sourceforge.net/wiki/index.php/Main_Page'
+description = """Gerris is a Free Software program for the solution of the partial
+                 differential equations describing fluid flow"""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+sources = ['gerris-snapshot-131206.tar.gz']
+source_urls = ['http://gerris.dalembert.upmc.fr/gerris/tarballs']
+checksums = ['d5346a362b104ccc858c4b79938727d56c4654b103c268e54cf3aa56d0b55b39']
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/g/Gerris/Gerris-20131206-foss-2019a.eb]: please remove trailing whitespace (fails style check)

File: easybuild/easyconfigs/g/Gerris/Gerris-20131206-foss-2019a.eb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+                 differential equations describing fluid flow"""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+sources = ['gerris-snapshot-131206.tar.gz']
+source_urls = ['http://gerris.dalembert.upmc.fr/gerris/tarballs']
+checksums = ['d5346a362b104ccc858c4b79938727d56c4654b103c268e54cf3aa56d0b55b39'] 
+
+dependencies = [
+    ('GTS', '20121130'),
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/g/Gerris/Gerris-20131206-foss-2019a.eb]: same as above

**Comment by boegel** [comment]
no response to requested changes for over 2 years, so closing this PR...

**Pull request closed without merging**